atom/browser/api/atom_api_web_contents.cc

```diff
@@ -27,6 +27,7 @@
 #include "atom/browser/child_web_contents_tracker.h"
 #include "atom/browser/lib/bluetooth_chooser.h"
 #include "atom/browser/native_window.h"
+#include "atom/browser/net/atom_navigation_throttle.h"
```

Comment:
Can you move the `atom_navigation_throttle` under `atom/browser`,  the throttler is entirely used on the UI thread and it doesn't directly handle any component from `//net` of chromium.